smarts/ros/README.md

```diff
@@ -27,7 +27,7 @@ Setup your environment:
 source /opt/ros/kinetic/setup.bash
 ```
 ```bash
-cd examples/ros
+cd smarts/ros
 catkin_make
```

Comment:
could mention ` python -m smarts.ros setup_node` here as an alternative?

Reply:
Hmm... interesting, I never even thought of that!  I originally only intended that script to be used after someone pip installs smarts, whereas these instructions were for people who have git cloned the repo.   So I never tested the script in this context.  However, upon looking at it, it should work for repo users too.

So I'll update the README now.  Good call!  Thanks.